tests/pausing_test.go

```diff
@@ -335,7 +335,7 @@ var _ = Describe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com][level:com
 
 			})
 
-			It("[test_id:3083]should gracefully handle console connection", func() {
+			It("[test_id:3083]should connect to serial console", func() {
```

Comment:
Please don't forget to update Polarion with the new headline as soon as the PR will be merged.

Reply:
Will do